run_ros1.sh

```
@@ -0,0 +1,34 @@
+CTRL_PID=$!
+
+# Start position commander
+echo "[INFO] Launching position commander..."
+./build/Release/position_command_node &
+CTRL_PID=$!
+
+sleep 5
+rosservice call /takeoff "{}"
+
+# === CLEANUP HANDLER ===
+trap "echo; echo '[INFO] Shutting down...'; kill $VOLASIM_PID $BRIDGE_PID $CTRL_PID; wait" SIGINT SIGTERM
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)